python_modules/dagster/dagster/_core/telemetry.py

```diff
@@ -75,7 +75,7 @@
 
 
 TELEMETRY_WHITELISTED_FUNCTIONS = {
-    "_logged_execute_pipeline",
+    "_logged_execute_job",
```

Comment:
I think this is fine - but mind making a change in the telemetry wiki that notes that `execute_pipeline_started` has been renamed?